Formula/grafana-cloud-agent.rb

```diff
@@ -0,0 +1,103 @@
+class GrafanaCloudAgent < Formula
+  desc "Lightweight subset of Prometheus and more, optimized for Grafana Cloud"
+  homepage "https://grafana.com/products/cloud/"
+  url "https://github.com/grafana/agent/archive/v0.8.0.tar.gz"
+  sha256 "3b8cb72793888f676fb205ced065da173429289dd29c77b79b095350ec28bf35"
+  license "Apache-2.0"
+
+  livecheck do
+    url "https://github.com/grafana/agent/releases/latest"
+    regex(%r{href=.*?/tag/v?(\d+(?:\.\d+)+)["' >]}i)
+  end
+
+  depends_on "go" => :build
+
+  def install
+    mkdir_p buildpath/"src/github.com/grafana"
+    ln_sf buildpath, buildpath/"src/github.com/grafana/agent"
+
+    system "make", "agent"
+
+    bin.install "./cmd/agent/agent"
+    mv "#{bin}/agent", "#{bin}/grafana-cloud-agent"
+
+    (bin/"grafana-cloud-agent_brew_services").write <<~EOS
+      #!/bin/bash
+      exec #{bin}/grafana-cloud-agent $(<#{etc}/grafana-cloud-agent.args)
+    EOS
```

Comment:
If this is not the default we shouldn't force it.

Reply:
Here I was following what the [Prometheus formula](https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/prometheus.rb#L38) is doing as well; is there a better way to pass default arguments to a launchd service?

Reply:
You can just add `--arg stuff` in the launchd file